raiwidgets/raiwidgets/_cohort.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+"""Module for defining cohorts in raiwidgets package."""
```

Comment:
I guess my biggest question here is whether this should live in raiwidgets package, responsibleai package, or be its own package.  The problem with it being in raiwidgets is that then it can't be re-used in, for example, erroranalysis, since raiwidgets depends on erroranalysis but not the other way around.  In case other packages (like RAI mitigations) want to re-use this, it will be more difficult if it is in raiwidgets package.

Reply:
I agree with all of the above. I do see that in future the cohort and filter definitions will go in separate package. Currently, for the pre-built cohort feature it is ok to keep it in raiwidgets. When this feature is complete we can think about moving these class definitions into a separate package and take dependency on it.